code/source/App.cpp

```diff
@@ -372,14 +372,11 @@ Point2 App::getViewDirection()
 	return Point2(az, el);
 }
 
-Point2 App::getTargetDirection()
```

Comment:
We could keep `App::getTargetDirection()` in it's old form and just add `App::getTargetPosition()` as implemented here. This gives us more flexibility to switch which function we use later.

Reply:
I agree with Josef's suggestion here. A reminder that this also requires changes in where the logging is done.